### PR TITLE
10 0 doc ide

### PR DESCRIPTION
**Pull request by TheEvilQueen-IDE**

### CODE REVIEW

**Comment by dbo-odoo** [comment]
@TheEvilQueen-IDE 
Made some changes, mostly:
- new screenshots (better quality - smaller size)
- changed section order (registration seems more important than duplication)
- unified structure (titles/subtitles/sections)
- some typos or weird phrases (at least weird to me)
- added some info when I felt it was appropriate

We'll have a look tomorrow if yu don't mind :)

File: db_management/db_premise.rst [hunk 1]
```diff
@@ -6,9 +6,171 @@
...
+If your database reaches its expiration date before your renew your subscription,
+you will encounter this message:
+
+.. image:: media/database_expired.png
+    :align: center
+    :alt: This database has expired.
+
+
+This **blocking** message appears after a non-blocking message that lasts 30 days.
+If you fail to take action before the end of the countdown,database is expires.
```
Comment (tivisse) [inline db_management/db_premise.rst]: Thx sir

**Comment by tivisse** [comment]
Landed in 10.0 at https://github.com/odoo/documentation-user/commit/c6b62deb7f61157deba0c4a6367e7da005f98517

The modifications will be online tomorrow.

Thanks for the clean PR.

**Comment by dbo-odoo** [comment]
Thx :postal_horn:

**Comment by dbo-odoo** [comment]
Mode poil de culteux:
si tu force-pushe ton rebase sur la feature branch avant de pusher sur la main branch, la PR est marquée comme mergée et non comme fermée :)

**Comment by dbo-odoo** [comment]
tellement de franglais, j'ai les yeux qui piquent!

**Comment by tivisse** [comment]
Je sais Damien. J'avais pas vu qu'il y avait une feature branch.

**Pull request closed without merging**